docs/developer-docs/latest/setup-deployment-guides/typescript.md

```diff
@@ -95,3 +125,116 @@ New plugins can be generated following the [plugins development documentation](/
 ::: note
 It is not necessary to repeat the `yarn install` or `npm run install` command after the initial installation. The `yarn build` or `npm run build` command is necessary to implement any plugin development that affects the admin panel.
 :::
+
+## Start Strapi programmatically
```

Comment:
I'm having a hard time understanding that whole section to be honest 🤔  I think we have several structural issues here. Let's discuss it together, also with @pwizla who also added quite a few comments on that section.